src/styles/blocks/card.scss

```
@@ -0,0 +1,136 @@
+.catalog {
+  display: grid;
+  margin: 0;
+
+  &__content {
+    display: grid;
+    margin: 0 auto;
+    padding: 50px 40px;
+    gap: 46px 48px;
+    grid-template-columns: repeat(1, $cardminwidth);
+
+    @media (min-width: 488px) {
+      & {
+        grid-template-columns: repeat(2, $cardminwidth);
+      }
+    }
```

what if you do like that?
```suggestion
    @media (min-width: 488px) {
      grid-template-columns: repeat(2, $cardminwidth);
    }
```